ui/Screen/Project.svelte

```diff
@@ -46,13 +51,13 @@
     selectPeer(peer);
   };
 
-  onMount(() => {
+  onDestroy(
```

Comment:
Why did you change this?

Reply:
Because we did not return anything from the callback to `onMount` we never stopped listening to the events. I could have added a return but felt that `onDestroy` is more straight-forward.

Reply:
Can we make this one more explicit as well?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1911/commits/6e2f1b544e8b6c807605520e572978995bd131fa